sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 ## 5.2.0-beta.5 (Unreleased)
 
 ### Features Added
+- Added `displayName` property to the following classes to set the optional display name:
```

Comment:
maybe specify that it is the display name for the job/operation?

Reply:
Looks like he should've copied my text after I made the change based on your same feedback on my PR. 😀